5 - months with days/solution_5.rb

```diff
@@ -9,7 +11,7 @@ def initialize(year)
 
   def generate_months_with_days_count
     months_with_days = {}
-    (1..12).each do |i|
+    (FIRST_MONTH..LAST_MONTH).each do |i|
```

Comment:
@alexshvedko fixed magic numbers. declared two constants. `FIRST_MONTH` and `LAST_MONTH`